nominal/core/client.py

```diff
@@ -818,6 +819,13 @@ def search_assets(
         """
         return list(self._iter_search_assets(search_text, label, property))
 
+    def create_data_review_batch_builder(self) -> DataReviewBatchBuilder:
```

Comment:
what's the `Batch` terminology for? also, in line with checklists, we should probs just use the `_builder` suffix and no `create_` prefix

Reply:
I used the Conjure naming, since I didn't have other context for naming.